### PR TITLE
[4.0] [plg_fields_subfields] Use namespaced class

### PR DESCRIPTION
**Pull request by SharkyKZ**
### Summary of Changes

Removes import pointing to non-existing file in favor of namespaced class.

### Testing Instructions

Code review is fine.

### Documentation Changes Required

No.

### CODE REVIEW

File: plugins/fields/subfields/subfields.php [hunk 1]
```diff
@@ -10,11 +10,10 @@
 use Joomla\CMS\Factory;
 use Joomla\CMS\Form\Form;
 use Joomla\Component\Fields\Administrator\Helper\FieldsHelper;
+use Joomla\Component\Fields\Administrator\Plugin\FieldsPlugin;
 
 defined('_JEXEC') or die;
```
Comment (Quy) [inline plugins/fields/subfields/subfields.php]: Move above line 10.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 4ffc3d426891d1539f02245c079b1246ee5edb4b<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27718">issues.joomla.org/tracker/joomla-cms/27718</a>.</sub>

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 4ffc3d426891d1539f02245c079b1246ee5edb4b<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27718">issues.joomla.org/tracker/joomla-cms/27718</a>.</sub>

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27718">issues.joomla.org/tracker/joomla-cms/27718</a>.</sub>

**Comment by richard67** [comment]
Drone failure seems not to be related to this PR.

**Comment by rdeutz** [comment]
@richard67 you are right the failure of drone was not realted to the PR, but then we have to restart drone to make sure it is not releated. My personal opinon is that nothing can be RTC when CI has failed.

**Comment by richard67** [comment]
@rdeutz Well, then tests should be more reliable. Currently it's a kind of lotto if they fail or not.

**Pull request merged**